stdlib/test/builtin/test_int.mojo

```diff
@@ -195,6 +195,54 @@ def test_int_uint():
     assert_equal(0, int(u2))
 
 
+def test_lt():
+    assert_true(Int(5).__lt__(Int(10)))
+    assert_true(Int(-10).__lt__(Int(-5)))
+    assert_false(Int(0).__lt__(Int(0)))
+    assert_false(Int(10).__lt__(Int(5)))
+
+
+def test_le():
+    assert_true(Int(5).__le__(Int(10)))
+    assert_true(Int(-10).__le__(Int(-5)))
+    assert_true(Int(0).__le__(Int(0)))
+    assert_false(Int(10).__le__(Int(5)))
+
+
+def test_eq():
+    assert_true(Int(5).__eq__(Int(5)))
+    assert_true(Int(0).__eq__(Int(0)))
+    assert_false(Int(0).__eq__(Int(1)))
+    assert_false(Int(5).__eq__(Int(10)))
+
+    var a: Int = 5
+    assert_true(a.__eq__(a))
```

Comment:
This case (and the one below for `__ne__`) is not really needed, could you please drop them?